src/main/java/macrobase/analysis/pipeline/Pipeline.java

```diff
@@ -0,0 +1,9 @@
+package macrobase.analysis.pipeline;
+
+import macrobase.analysis.result.AnalysisResult;
+import macrobase.conf.MacroBaseConf;
+
+public interface Pipeline {
+    void initialize(MacroBaseConf conf) throws Exception;
```

Comment:
If we have this return a Pipeline we can do stuff like (new Pipeline()).initialize(x).run() on a single line which is nice.